tenet/assetlist.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "tenet",
+  "assets": [
+    {
+      "description": "The native EVM, governance and staking token of the Tenet",
+      "denom_units": [
+        {
+          "denom": "atenet",
+          "exponent": 0
+        },
+      ],
+      "base": "atenet",
+      "name": "Tenet",
+      "display": "tenet",
```

Comment:
this display [denom] property is mean to point to a defined denom_unit which relays the number of decimal points to use. If the practically used unit TENET has zero exponent (is indivisible), then the display should be atenet, but I'm guessing from the 'a' that there should be another denom_unit called 'tenet' with exponent 18...